Lib/test/test_string_literals.py

```diff
@@ -266,6 +266,13 @@ def test_eval_str_u(self):
         self.assertRaises(SyntaxError, eval, """ bu'' """)
         self.assertRaises(SyntaxError, eval, """ ub'' """)
 
+    def test_uppercase_prefixes(self):
+        self.assertEqual(eval(""" B'x' """), b'x')
+        self.assertEqual(eval(r""" R'\x01' """), '\\x01')
+        self.assertEqual(eval(r""" BR'\x01' """), b'\\' + b'x01')
+        self.assertEqual(eval(""" F'{1+1}' """), '2')
+        self.assertEqual(eval(""" U'\U0001d120' """), '\U0001d120')
```

Comment:
Presumably the lowercase versions are covered by other tests (right?)

Would it be simpler here to just assert in each case that the lowercase and upper case prefixes generate the same thing?  For instance, ``self.assertEqual(eval(""" F'{1+1}' """), eval(""" f'{1+1}' """))``.